common/app/common/dfp/SurveySponsorshipAgent.scala

```diff
@@ -0,0 +1,32 @@
+package common.dfp
+
+import play.api.mvc.RequestHeader
+import model.{MetaData}
+import model.DotcomContentType
+import conf.switches.Switches.{SurveySwitch}
```

Comment:
This Switch was originally implemented to allow us (or whoever!) to prevent the survey slot from being added to the article if a campaign was not running. Now we have a fancy automated agent to do this 😍 Do we still need this switch?

Reply:
Have made a separate card for removing the switch 👍